renderlib/io/FileReaderZarr.cpp

```diff
@@ -143,11 +171,57 @@ FileReaderZarr::getChannelNames(const std::string& filepath)
   }
   return channelNames;
 }
+
+nlohmann::json
+FileReaderZarr::getMultiscales(nlohmann::json attrs)
+{
+  nlohmann::json multiscales;
+  if (m_zarrVersion == 3) {
+    auto attributes = attrs["attributes"];
+    if (attributes.is_null()) {
+      LOG_ERROR << "No attributes found in zarr.json";
+      return multiscales;
```

Comment:
Is `multiscales` a nullptr in this return statement?

Reply:
basically, yeah, it's a default-initialized empty nlohmann::json  object. (this is an error)